### PR TITLE
Java: Add `SELECT()` command for standalone

### PR DESCRIPTION
**Pull request by acarbonetto**

### CODE REVIEW

File: java/client/src/main/java/glide/api/commands/ServerManagementCommands.java [hunk 1]
```diff
@@ -32,4 +32,13 @@ public interface ServerManagementCommands {
      *     sections requested.
      */
     CompletableFuture<String> info(InfoOptions options);
+
+    /**
+     * Change the currently selected Redis database.
+     *
+     * @see <a href="https://redis.io/commands/select/">redis.io</a> for details.
+     * @param index - The index of the database to select.
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/commands/ServerManagementCommands.java:40]:
```suggestion
     * @param index The index of the <code>database</code> to select.
```

File: java/client/src/main/java/glide/api/models/Transaction.java [hunk 2]
```diff
@@ -27,4 +30,18 @@ public class Transaction extends BaseTransaction<Transaction> {
     protected Transaction getThis() {
         return this;
     }
+
+    /**
+     * Change the currently selected Redis database.
+     *
+     * @see <a href="https://redis.io/commands/select/">redis.io</a> for details.
+     * @param index - The index of the database to select.
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/models/Transaction.java:38]:
```suggestion
     * @param index The index of the <code>database</code> to select.
```

**Review by Yury-Fridlyand — APPROVED** [review]

File: java/client/src/main/java/glide/api/models/Transaction.java (hunk 2 above)
Comment (acarbonetto) [reply java/client/src/main/java/glide/api/models/Transaction.java:38]: why do you want to emphasize database?

File: java/client/src/main/java/glide/api/commands/ServerManagementCommands.java (hunk 1 above)
Comment (acarbonetto) [reply java/client/src/main/java/glide/api/commands/ServerManagementCommands.java:40]: why do you want to emphasize database?

Comment (Yury-Fridlyand) [reply java/client/src/main/java/glide/api/commands/ServerManagementCommands.java:40]: It is one the redis terms, as set (not a command name)

Comment (acarbonetto) [reply java/client/src/main/java/glide/api/commands/ServerManagementCommands.java:40]: We don't emphasize those terms.  Just Java terms.  Same goes for 'index'.  For Redis-based set, we wouldn't emphasize it either.  Although, we could...

**Pull request merged**